lib/tasks/locale.rake

```diff
@@ -141,12 +141,18 @@ namespace :locale do
     system({"RAILS_ENV" => "i18n"}, "bundle exec rake locale:store_model_attributes")
   end
 
+  task "delete_pot_file", :root do |_, args|
+    pot_file = Dir.glob("#{args[:root]}/locale/*.pot").first
+    system('rm', '-vf', pot_file) if pot_file
```

Comment:
I'm fine with this, but is it better to just do `FileUtils.rm_f(pot_file) if pot_file`?

Reply:
Yep, you're right. Fixed.